django/db/models/functions/math.py

```diff
@@ -0,0 +1,117 @@
+from django.db.models import Func, Transform, fields
+
+
+class Abs(Transform):
+    function = 'ABS'
+    lookup_name = 'abs'
+
+
+class Acos(Transform):
+    function = 'ACOS'
+    lookup_name = 'acos'
+    output_field = fields.FloatField()
+
+
+class Asin(Transform):
+    function = 'ASIN'
+    lookup_name = 'asin'
+    output_field = fields.FloatField()
+
+
+class Atan(Transform):
+    function = 'ATAN'
+    lookup_name = 'atan'
+    output_field = fields.FloatField()
+
+
+class Atan2(Func):
+    function = 'ATAN2'
+    arity = 2
+    output_field = fields.FloatField()
+
+
+class Ceiling(Transform):
+    function = 'CEILING'
+    lookup_name = 'ceiling'
+
+
+class Cos(Transform):
+    function = 'COS'
+    lookup_name = 'cos'
+    output_field = fields.FloatField()
+
+
+class Cot(Transform):
+    function = 'COT'
+    lookup_name = 'cot'
+    output_field = fields.FloatField()
+
+
+class Degrees(Transform):
+    function = 'DEGREES'
+    lookup_name = 'degrees'
+    output_field = fields.FloatField()
+
+
+class Exp(Transform):
+    function = 'EXP'
+    lookup_name = 'exp'
+
+
+class Floor(Transform):
+    function = 'FLOOR'
+    lookup_name = 'floor'
+
+
+class Log(Transform):
+    function = 'LOG'
+    lookup_name = 'log'
+
+    def as_postgresql(self, compiler, connection):
+        return super().as_sql(compiler, connection, function='LN')
+
+
+class Mod(Func):
+    function = 'MOD'
+    arity = 2
+    output_field = fields.FloatField()
+
+
+class Pi(Func):
+    function = 'PI'
+    arity = 0
+    output_field = fields.FloatField()
+
+
+class Power(Func):
+    function = 'POWER'
+    arity = 2
+    output_field = fields.FloatField()
+
+
+class Radians(Transform):
+    function = 'RADIANS'
+    lookup_name = 'radians'
+    output_field = fields.FloatField()
+
+
+class Round(Transform):
```

Comment:
I haven't had time to look, but we should see what additional arguments `ROUND()` can take in different backends.

Reply:
Indeed round also behaves differently in different database. For example, in mysql ROUND(X), ROUND(X,D): Rounds the argument X to D decimal places. The rounding algorithm depends on the data type of X. D defaults to 0 if not specified. 

In oracle ROUND returns n rounded to integer places to the right of the decimal point. If you omit integer, then n is rounded to 0 places.

Reply:
IMO it works exactly the same way in [Oracle](https://docs.oracle.com/database/121/SQLRF/functions169.htm#SQLRF00698) and [MySQL](https://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_round) database, am I missing sth? 🤔

Reply:
Maybe it could be generalized by always passing the explicit argument `D` to 0, if not called differently.